Dockerfile

```diff
@@ -0,0 +1,57 @@
+ARG TOMCAT_VERSION=9.0.38
+ARG JAVA_VERSION=8
```

Comment:
I would stick to Java 11 which has Marlin built-in to save some hassle

Reply:
I switched to OpenJDK 11 in a03ee5a